lib/vagrant-libvirt/util/network_util.rb

```diff
@@ -97,14 +105,18 @@ def configured_networks(env, logger)
             # store type in options
             # use default values if not already set
             options = {
-              iface_type:  type,
-              netmask:      '255.255.255.0',
+              iface_type:   type,
+              netmask:      options[:libvirt__network_address] ?
+                            IPAddr.new(options[:libvirt__network_address]).get_mask :
```

Comment:
Looks like we can safely use `:network_address` here and similarly below `:network_name` instead of `:libvirt__network_address` and `:libvirt__network_name` as options is provided by `scoped_hash_override(original_options, :libvirt)`.